pipeline_preprocess.sh

```diff
@@ -141,4 +153,33 @@ echo 'normalizing ...'
 bcftools ${BCFTOOLS_ARGS}
 echo 'normalizing done'
 
+REMOVE_ANN_OUTPUT_DIR="${OUTPUT_DIR_ABSOLUTE}"/step1_remove_annotations
```

Comment:
Annotations should be removed before normalization so that the normalized VCF file is valid in case you run the pipeline with `--keep`.